recipe/meta.yaml

```diff
@@ -5,20 +5,23 @@ package:
   version: {{ version }}
 
 source:
-  url: https://www.cl.cam.ac.uk/~mgk25/jbigkit/download/jbigkit-{{ version }}.tar.gz
-  sha256: de7106b6bfaf495d6865c7dd7ac6ca1381bd12e0d81405ea81e7f2167263d932
+  - url: https://www.cl.cam.ac.uk/~mgk25/jbigkit/download/jbigkit-{{ version }}.tar.gz
+    sha256: de7106b6bfaf495d6865c7dd7ac6ca1381bd12e0d81405ea81e7f2167263d932
+  - path: CMakeLists.txt
 
 build:
-  number: 2001
-  skip: true  # [win]
+  number: 2002
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - {{ compiler('c') }}  # [not win]
+    - vs2017_win-64        # [win]
```

Comment:
Why is this necessary?

Reply:
@nehaljwani By default, it picks up the vs2015 on Windows. However somehow I just cannot get it built on the cloud pipeline and a mysterious error throw to me (see below). So I gave it a try to the newer toolchain and the problem is just gone.

```
[100%] Linking C shared library libjbig.dll
NMAKE : fatal error U1077: 'D:\bld\jbig_1585696984760\_build_env\Library\bin\cmake.exe' : return code '0xc0000005'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.

```

Reply:
You can remove this change and rerender in an hour. See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/501

Reply:
@isuruf Hooray! And just a quick question, as the default is going to be migrated to vs2017, is there a plan to start adding `vs2019_win-64` for preview.

Reply:
PRs welcome